Gemfile

```diff
@@ -10,6 +10,7 @@ ruby "2.4.1"
 gem "bcrypt"
 gem "bourbon"
 gem "delayed_job_active_record"
+gem "enum_help"
```

Comment:
Rails has in-built support for Enums now. So no need to use an external gem for this functionality.
Have a look at the curriculum for Enums and how to implement them

Reply:
Thanks, done in sha: c751e98